smithy-docgen/src/main/java/software/amazon/smithy/docgen/core/DirectedDocGen.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.docgen.core;
```

Comment:
The `core` suffix doesnt seem necessary to me. Could we just have this be `software.amazon.smithy.docgen`?